axonserver/src/main/java/io/axoniq/axonserver/grpc/AuthenticationInterceptor.java

```diff
@@ -41,23 +41,26 @@ public <T, R> ServerCall.Listener<T> interceptCall(ServerCall<T, R> serverCall,
         String token = token(metadata);
         String context = context(metadata);
         StatusRuntimeException sre = null;
-
+        Authentication authentication = null;
         if (token == null) {
             AuditLog.getLogger().warn("{}: Request without token sent from {}",
                                       context,
                                       serverCall.getAttributes().get(Grpc.TRANSPORT_ATTR_REMOTE_ADDR));
             sre = GrpcExceptionBuilder.build(ErrorCode.AUTHENTICATION_TOKEN_MISSING,
                                              "No token for " + serverCall.getMethodDescriptor().getFullMethodName());
-        } else if (!axonServerAccessController.allowed(serverCall.getMethodDescriptor().getFullMethodName(),
-                                                       context,
-                                                       token)) {
-            AuditLog.getLogger().warn("{}: Request with invalid token for {} sent from {}",
-                                      context,
-                                      serverCall.getMethodDescriptor().getFullMethodName(),
-                                      serverCall.getAttributes().get(Grpc.TRANSPORT_ATTR_REMOTE_ADDR));
-            sre = GrpcExceptionBuilder.build(ErrorCode.AUTHENTICATION_INVALID_TOKEN,
-                                             "Invalid token for " + serverCall.getMethodDescriptor()
-                                                                              .getFullMethodName());
+        } else {
+            authentication = authentication(token);
```

Comment:
Readability/maintainability: Shouldn't the method be a verb, like `authenticate` or `validate` or even a verb-phrase `isValid`? Currently this line is weird/puzzling.